runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -102,72 +107,12 @@ pub mod utils {
 		}
 	}
 
-	pub fn setup_xcm<T: Runtime + FudgeSupport>(env: &mut FudgeEnv<T>) {
```

Comment:
Note: Moved here: https://github.com/centrifuge/centrifuge-chain/pull/1833/files#diff-3baaa18fad9b36a69774c73a025373847f78da6ff3458c3a088275ec1ee59f84

Reply:
Yeah! Sorry for not indicating. `proxy.rs` also requires so it should be common to not link cases againts other cases